internal/resource/configmap_test.go

```diff
@@ -252,13 +296,66 @@ listeners.ssl.default = 5671
 ssl_options.certfile   = /etc/rabbitmq-tls/tls.crt
 ssl_options.keyfile    = /etc/rabbitmq-tls/tls.key
 listeners.ssl.default  = 5671
+
+management.ssl.certfile   = /etc/rabbitmq-tls/tls.crt
+management.ssl.keyfile    = /etc/rabbitmq-tls/tls.key
+management.ssl.port       = 15671
+
 ssl_options.cacertfile = /etc/rabbitmq-tls/ca.certificate
 ssl_options.verify     = verify_peer
+management.ssl.cacertfile = /etc/rabbitmq-tls/ca.certificate
 `)
 
 				Expect(configMapBuilder.Update(configMap)).To(Succeed())
 				Expect(configMap.Data).To(HaveKeyWithValue("rabbitmq.conf", expectedRabbitmqConf))
 			})
+
+			When("Web MQTT and Web STOMP are enabled", func() {
+				It("adds TLS config for the additional plugins", func() {
+					instance = rabbitmqv1beta1.RabbitmqCluster{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "rabbit-tls",
+						},
+						Spec: rabbitmqv1beta1.RabbitmqClusterSpec{
+							TLS: rabbitmqv1beta1.TLSSpec{
+								SecretName:   "tls-secret",
+								CaSecretName: "tls-mutual-secret",
+								CaCertName:   "ca.certificate",
+							},
+							Rabbitmq: rabbitmqv1beta1.RabbitmqClusterConfigurationSpec{
+								AdditionalPlugins: []rabbitmqv1beta1.Plugin{
+									"rabbitmq_mqtt",
+									"rabbitmq_stomp",
+									"rabbitmq_web_mqtt",
+									"rabbitmq_web_stomp",
+									"rabbitmq_amqp_1_0",
+								},
+							},
+						},
+					}
+
+					expectedRabbitmqConf := iniString(defaultRabbitmqConf(builder.Instance.Name) + `
+ssl_options.certfile   = /etc/rabbitmq-tls/tls.crt
+ssl_options.keyfile    = /etc/rabbitmq-tls/tls.key
+listeners.ssl.default  = 5671
+
+management.ssl.certfile   = /etc/rabbitmq-tls/tls.crt
+management.ssl.keyfile    = /etc/rabbitmq-tls/tls.key
+management.ssl.port       = 15671
+
+mqtt.listeners.ssl.default = 8883
+
+stomp.listeners.ssl.1 = 61614
+
+ssl_options.cacertfile = /etc/rabbitmq-tls/ca.certificate
+ssl_options.verify     = verify_peer
+management.ssl.cacertfile = /etc/rabbitmq-tls/ca.certificate
+`)
```

Comment:
How are we testing Web STOMP and Web MQTT here?

Following [RabbitMQ docs](https://www.rabbitmq.com/web-stomp.html#tls), I was expecting to see the keys `web_stomp.ssl.certfile` and `web_stomp.ssl.keyfile`. Same story for [MQTT](https://www.rabbitmq.com/web-mqtt.html#tls).

Reply:
Web mqtt and web stomp are not implemented yet. The decision to add them too was reached at sync-up and is one of the actions (along with removing the tls property) that we need to do.

Reply:
Hi @Zerpet. configmap additions for web_mqtt and web_stomp have been added here https://github.com/rabbitmq/cluster-operator/pull/451/commits/bc28103a2ba49af6a2d62e6facc1cf054203357c. Please take a look and let me know if it looks right.